### ISSUE TITLE
Problems with node features

### ISSUE THREAD

**Issue by yxiong2023** [qbmizsj/A-GCL#2]
I wonder if node features make a significant contribution to the final experimental results. Have you ever used FC vectors as node features?

**Comment by qbmizsj** [comment]
No. How could FC vectors be viewed as node features? Since FC denotes the weighted adjacent matrix. And what is the meaning of this operation?&nbsp;



发自我的iPhone


------------------ Original ------------------
From: UP ***@***.***&gt;
Date: Tue,Apr 9,2024 3:38 PM
To: qbmizsj/A-GCL ***@***.***&gt;
Cc: Subscribed ***@***.***&gt;
Subject: Re: [qbmizsj/A-GCL] Problems with node features (Issue #2)





 
I wonder if node features make a significant contribution to the final experimental results. Have you ever used FC vectors as node features?

**Comment by yxiong2023** [comment]
> No. How could FC vectors be viewed as node features? Since FC denotes the weighted adjacent matrix. And what is the meaning of this operation?&nbsp; 发自我的iPhone
> […](#)
> ------------------ Original ------------------ From: UP ***@***.***&gt; Date: Tue,Apr 9,2024 3:38 PM To: qbmizsj/A-GCL ***@***.***&gt; Cc: Subscribed ***@***.***&gt; Subject: Re: [qbmizsj/A-GCL] Problems with node features (Issue #2) I wonder if node features make a significant contribution to the final experimental results. Have you ever used FC vectors as node features? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&gt;

Thank you for your reply. I mean, just treat each row of the FC matrix as the feature vector of each node.
